src/main/app/case/case.ts

```diff
@@ -64,10 +64,12 @@ export interface Case {
   caseCode?: string;
   accessCode?: string;
   detailsKnown?: string;
+  miam_otherProceedings?: string;
   miam_haveDocSigned?: string;
   startAlternative?: string;
   contactDetailsPrivate?: string;
   miamStart?: string;
+  miamConsent?: string;
```

Comment:
@rahulsanjava19 Please rename this field to 'miam_consent'
If this is existing field then define a new one & don't alter the existing.